### PR TITLE
style: textarea should avtive when focused

### PR DESCRIPTION
**Pull request by MuxinFeng**
[[中文版模板 / Chinese template](https://github.com/ant-design/ant-design/blob/master/.github/PULL_REQUEST_TEMPLATE/pr_cn.md?plain=1)]

### 🤔 This is a ...

- [ ] New feature
- [ ] Bug fix
- [ ] Site / documentation update
- [ ] Demo update
- [x] Component style update
- [ ] TypeScript definition update
- [ ] Bundle size optimization
- [ ] Performance optimization
- [ ] Enhancement feature
- [ ] Internationalization
- [ ] Refactoring
- [ ] Code style optimization
- [ ] Test Case
- [ ] Branch merge
- [ ] Workflow
- [ ] Other (about what?)

### 🔗 Related issue link

https://github.com/ant-design/ant-design/issues/41164

### 📝 Changelog

| Language   | Changelog |
| ---------- | --------- |
| 🇺🇸 English |     textarea should avtive when focused      |
| 🇨🇳 Chinese |     textarea 应该在聚焦时保持高亮      |

### ☑️ Self-Check before Merge

⚠️ Please check all items below before requesting a reviewing. ⚠️

- [x] Doc is updated/provided or not needed
- [x] Demo is updated/provided or not needed
- [x] TypeScript definition is updated/provided or not needed
- [x] Changelog is provided or not needed

### CODE REVIEW

**Comment by MadCcc** [comment]
ci 挂了。4.x 和 master 有差异，注意更新下本地依赖。

**Comment by MuxinFeng** [comment]
> ci 挂了。4.x 和 master 有差异，注意更新下本地依赖。

大佬，我之前试过把 node_modules 和 yarn.lock 都删掉重装，但还是过不了测试，而且那部分测试都不是 input 组件这里的。

**Comment by yoyo837** [comment]
Rebase 一下

**Comment by MuxinFeng** [comment]
目前本地运行看不出错来，只能根据 ci 的报错慢慢调整了

**Comment by MuxinFeng** [comment]
emmm，ci 跟 本地报错一直不能统一，暂时搁置一下啦

File: components/input/ClearableLabeledInput.tsx [hunk 1]
```diff
@@ -56,7 +56,7 @@ class ClearableLabeledInput extends React.Component<ClearableInputProps> {
         onClick={handleReset}
         // Do not trigger onBlur when clear input
         // https://github.com/ant-design/ant-design/issues/31200
-        onMouseDown={e => e.preventDefault()}
+        onMouseDown={(e) => e.preventDefault()}
```
Comment (yoyo837) [inline components/input/ClearableLabeledInput.tsx:59]: 这种改动恢复吧

Comment (MuxinFeng) [reply components/input/ClearableLabeledInput.tsx:59]: 11

**Review by yoyo837 — APPROVED** [review]

File: components/input/ClearableLabeledInput.tsx (hunk 1 above)
Comment (yoyo837) [reply components/input/ClearableLabeledInput.tsx:59]: 这个又回来了

Comment (MuxinFeng) [reply components/input/ClearableLabeledInput.tsx:59]: 嗯嗯，大佬好快

**Review by afc163 — APPROVED** [review]

File: components/input/ClearableLabeledInput.tsx (hunk 1 above)
Comment (MuxinFeng) [reply components/input/ClearableLabeledInput.tsx]: 好啦~

Comment (MuxinFeng) [reply components/input/ClearableLabeledInput.tsx]: 我这关了自动格式化都不管用，得走commit --no-verify才行，一不留神就格式化了。

File: components/input/__tests__/textarea.test.tsx [hunk 2]
```diff
@@ -574,4 +574,14 @@ describe('TextArea allowClear', () => {
 
     textareaSpy.mockRestore();
   });
+
+  it('should have correct className when onFocus and onBlur', () => {
+    const { container } = render(<TextArea allowClear />);
+
+    fireEvent.focus(container.querySelector('textarea')!);
+    expect(document.querySelector('.ant-input-affix-wrapper-focused')).toBeTruthy();
```
Comment (yoyo837) [inline components/input/__tests__/textarea.test.tsx:582]: 改container查询吧?

Comment (MuxinFeng) [reply components/input/__tests__/textarea.test.tsx]: 改好啦，本地跑了下测试没问题~

**Pull request merged**